internal/controller/drplacementcontrol.go

```diff
@@ -339,11 +339,16 @@ func (d *DRPCInstance) RunFailover() (bool, error) {
 	const done = true
 
 	if d.instance.Spec.FailoverCluster == "" {
-		msg := "missing value for spec.FailoverCluster"
-		addOrUpdateCondition(&d.instance.Status.Conditions, rmn.ConditionAvailable, d.instance.Generation,
-			d.getConditionStatusForTypeAvailable(), string(d.instance.Status.Phase), msg)
-
-		return done, fmt.Errorf(msg)
+		addOrUpdateCondition(
+			&d.instance.Status.Conditions,
+			rmn.ConditionAvailable,
+			d.instance.Generation,
+			d.getConditionStatusForTypeAvailable(),
+			string(d.instance.Status.Phase),
+			"missing value for spec.FailoverCluster",
+		)
+
+		return done, fmt.Errorf("missing value for spec.FailoverCluster")
```

Comment:
This uses constant string in Errorf, we can use errors.New()

Reply:
Addressing this in a refresh (and the other comment on the same line)

Reply:
Ugh! this is not simple. We import `github.com/pkg/errors` in this file and in a few others. Also this file imports `"k8s.io/apimachinery/pkg/api/errors"` as is, so about 10-12 instances use errors.IsXXX functions.

To use `errors..New()` it would need that we import "errors", import k8s errors as <foo> (typically `k8serrors`), change all instances to <foo> to keep it clean.

This is a lot for this PRs intention and/or to optimize this fmt call. Issue #1678 opened to tackle this cleanup.

Reply:
Sure we can do this later.